Refs #31169 -- Added release note for parallel test running changes. (#15599)
django/django: docs/releases/4.1.txt

  :meth:`~.SimpleTestCase.assertFormsetError` now support passing a
  form/formset object directly.

> * The :class:`.DiscoverRunner` now supports running tests in parallel on
>   systems, such as macOS and Windows, where the ``spawn``
>   :mod:`multiprocessing` start method is used.

Reviewer: A little word smithing...

```suggestion
* The :class:`.DiscoverRunner` now supports running tests in parallel on
  macOS, Windows, and any other systems where the default
  :mod:`multiprocessing` start method is ``spawn``.
```

Could also go as the first point in the section? Much bigger news than the others!

Author: Done. Thanks! 👍
